src/containers/Article.js

```diff
@@ -540,8 +532,8 @@ class Article extends PureComponent {
           <div className="hidden-print">
           </div>
         </div>
-        <MobileArticleTools topicTitle={topicTitle} topicSlug={topicSlug} isMobileToolsDisplayed={isMobileToolsDisplayed}/>
-        <DesktopArticleTools topicTitle={topicTitle} topicSlug={topicSlug} isDesktopToolsDisplayed={isDesktopToolsDisplayed} />
+        <MobileArticleTools ref={ ele => this.mat = ele } topicTitle={topicTitle} topicSlug={topicSlug} isMobileToolsDisplayed={false}/>
+        <DesktopArticleTools ref={ ele => this.dat = ele} topicTitle={topicTitle} topicSlug={topicSlug} isDesktopToolsDisplayed={false} />
```

Comment:
Can we just set initial state `{ toShow: false }` in `DesktopArticleTools` and won't need prop `isDesktopToolsDisplayed` anymore?